scripts/publish-environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+MY_PATH=$(dirname "$0")
+MY_PATH=$( cd "$MY_PATH" && pwd )
+
+DOCKER_IMAGES=$(docker image ls -qaf reference="$BEE_IMAGE_PREFIX/$BEE_ENV_PREFIX*:$BEE_VERSION")
+for DOCKER_IMAGE in $DOCKER_IMAGES
+do
+  echo "$DOCKER_IMAGE"
+  docker push "$DOCKER_IMAGE"
```

Comment:
Hmm, if I understand the images will be pushed with a tag similar to `$BEE_IMAGE_PREFIX/$BEE_ENV_PREFIX*:$BEE_VERSION` right?

I am not sure if this tag scheme is sufficient for our use case as I can imagine we might have multiple versions for given `BEE_VERSION`, which is then question if we pronounce these tags as "immutable" or "mutable". If mutable then there is no issue, but honestly, I am not a big fan of that as it is hard to reason what version are you using.

Reply:
you got my concerns, I think the docker pushes can act in a mutable way, but I'm not sure about the pulls...

Reply:
as I notice on the CI jobs here, it always pulls the docker images so it shouldn't cause problem on CI level